miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/view/EditProjectController.java

```diff
@@ -82,7 +82,7 @@ public ModelAndView byProjectShortName(@PathVariable String shortName, ModelMap
     if (project == null) {
       throw new NotFoundException("No project found for short name " + shortName);
     }
-    return new ModelAndView("forward:/project/code/%s".formatted(shortName), model);
+    return new ModelAndView("redirect:/project/code/%s".formatted(shortName), model);
```

Comment:
Everything above this line should be removed from the method (validation already happens after redirect)